### DISCUSSION TITLE
DULCE TENTACIÓN(PASTELES)

### DISCUSSION THREAD

**Discussion by floressdi** [Launch-X-Latam/MisionFrontEnd#307] [Q&A]
## Dulce Tentación (Pasteles)
https://floressdi.github.io/Dulce-Tentaci-n.floressdi/
PD:¿Cuantas practicas son de mision FrontEnd?

<img src="https://media1.giphy.com/media/cLpqzmqFc9u3aggR75/giphy.gif"/>

**Comment by JoseEduardoVelazquezCN** [comment]
Ammm técnicamente es una práctica por cada módulo: HTML, CSS y ahora que vamos con JS 

Y va quedando muy bien tu práctica 👏

**Reply by floressdi** [reply]
Muchas Gracias!!

**Comment by BlinterDev** [comment]
Increible página amigo! Amé el efecto de botón y la calidad de las imágenes 👏👏

**Reply by floressdi** [reply]
Muchas Gracias!!

**Comment by fabiancgc12** [comment]
4 practicas, la tercera es la de css que es clonar una pagina y la cuarta es la js de crear una pokedex

**Reply by floressdi** [reply]
Muchas Gracias!!
